### ISSUE TITLE
Make dispatch operator smart

### ISSUE THREAD

**Issue by AlexanderShushunov** [redneckz/react-redux-rxjs#2]
If we have an observable with action creators we should write something like this:
```
actionsCreators$::dispatch(actionCreator => actionCreator())
``` 
`dispatch` can recognize this case: dispatch does not have any arguments.
Observable value should be a function. If value is not a function warning should be emitted.

**Comment by redneckz** [comment]
Observable value should be a function. If value is not a function warning should be emitted.
Why warning? For example, Observable value can be an acion itself.

**Comment by redneckz** [comment]
I think that warning should be arised in case of function (action creator) in stream and dispatch operator supplied with action creator as first argument.

**Comment by redneckz** [comment]
Fixed in 103f4968a8555526a2e713a629e758a7cda87b0b
